CodeStarterCamp_Week2/LottoWinningChecker.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  LottoNumberChecker.swift
+//  CodeStarterCamp_Week2
+//
+//  Created by JIWOONG on 2023/08/01.
+//
+
+import Foundation
+
+// 당첨번호를 생성하는 함수
+func makeLottoNumber() -> Array<Int> {
```

Comment:
```suggestion
func makeLottoNumber() -> [Int] {
```
이렇게 표현하는 게 더 간결해서 저는 개인적으로 이쪽을 선호하는 편입니다 😀

Reply:
알려주신 방법이 더 쓰기 간편한 느낌입니다!